mindmeld/models/taggers/memm.py

```diff
@@ -175,7 +175,7 @@ def _predict_proba_distribution_example(self, example, config, resources):
             predictions.append(list(prediction))
             tag_maps.append(
                 [
-                    self.class_encoder.inverse_transform(i) for i in range(len(prediction))
+                    self.class_encoder.inverse_transform([i]) for i in range(len(prediction))
```

Comment:
Going by the changes in line 153, should this be as follows?
```suggestion
                    self.class_encoder.inverse_transform([i])[0] for i in range(len(prediction))
```

Reply:
You're right! I've fixed that! Thanks Varsha!